server/controllers/RequestController.js

```diff
@@ -19,10 +19,15 @@ export const getOneRequest = async (req, res) => {
 };
 
 export const getAllRequests = async (req, res) => {
+  const USER_ID = "660ce122f99a93e263f053b4";
```

Comment:
once userId is attached to header, get it from header instead of hardcode

Reply:
Done